private/tools/java/com/github/bazelbuild/rules_jvm_external/maven/MavenPublisher.java

```diff
@@ -106,6 +111,17 @@ public static void main(String[] args)
         futures.add(upload(repo, credentials, coords, "-javadoc.jar", docJar, gpgSign));
       }
 
+      if(args.length > 9) {
+        String ext = com.google.common.io.Files.getFileExtension(args[9]);
```

Comment:
Won't this value need to be calculated per tuple, rather than just on the final one? eg `war=foo.jar,tar=foo.tar` would result in `ext` being set to `tar` for both the `war` and `tar` classifiers.